schema/crdb/dbinit.sql

```diff
@@ -2131,17 +2140,7 @@ CREATE TABLE IF NOT EXISTS omicron.public.ip_pool_resource (
 
     -- resource_type is redundant because resource IDs are globally unique, but
     -- logically it belongs here
-    PRIMARY KEY (ip_pool_id, resource_type, resource_id),
-
-    -- Check that there are no default pools for the internal silo
-    CONSTRAINT internal_silo_has_no_default_pool CHECK (
-        NOT (
-            resource_type = 'silo' AND
-            resource_id = '001de000-5110-4000-8000-000000000001' AND
-            is_default
```

Comment:
Is removing this constraint correct as a part of this PR? I'm not sure how `reservation_type` / multiple external pools interacts with `is_default`. Will we let people set a default external service pool?